lib/cleanupDone.js

```diff
@@ -0,0 +1,17 @@
+function cleanupDone(config, sftp) {
+  var root = config.fileDownloadDir + '/' + config.completedDir
+  return sftp.list(root)
+    .then(files => {
+      return Promise.all(files.filter(file => {
+        const age = Date.now() - file.modifyTime
+        const keep = 1000 * config.fileRetentionDays * 60 * 60 * 24
+        return file.type == '-' && (age > keep)
```

Comment:
Is this some magic SFTP thing? I'm not sure what `file.type == '-'` means...

Reply:
Yeah that means it's a file. It's used in a few places. I think the `ssh2-sftp-client` library sets that.